wgsl/index.bs

```diff
@@ -4867,7 +4867,6 @@ I've written what an NVIDIA GPU does.  See https://github.com/google/amber/pull/
   <tr><td>`BLOCK`<td>block
   <tr><td>`BREAK`<td>break
   <tr><td>`CASE`<td>case
-  <tr><td>`CONST`<td>const
```

Comment:
Let's reserve `const`

Reply:
Yep, I replaced `let` with `const` below in the reserved words table.